crates/sallyport/tests/sallyport_enarxcall_tests.rs

```diff
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 
```

Comment:
I think this file should just be called `enarxcall.rs` for clearer test output. The `sallyport_` prefix and `_tests` suffix are redundant, because we're in a `tests` module within `sallyport` crate

Reply:
Ah, but check out this branch and try running it yourself. In the output of `cargo test --workspace`, it actually gives no indication at all that this test has anything to do with the sallyport crate. This is especially pernicious in the case of the syscall integration test, since both Enarx and Sallyport have a test with the same name. This is why I added these prefixes, despite the redundancy. I think making test output easier to read is worth it. Do you agree?

Reply:
Example output:

```
     Running tests/sallyport_enarxcall_tests.rs (/home/ben/code/enarx/target/debug/deps/sallyport_enarxcall_tests-7e9a637e36652c11)

running 9 tests
test cpuid ... ok
test balloon_memory ... ok
test remove_sgx_pages ... ok
test trim_sgx_pages ... ok
test mem_info ... ok
test get_sgx_target_info ... ok
test get_snp_vcek ... ok
test reset_sgx_permissions ... ok
test get_sgx_quote ... ok

test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

Note that the only place that "sallyport" shows up in the output at all is via the file name here.

Reply:
I could also suggest that this is a bug in Cargo, and that the first line of that output should read like:

```
     Running crates/sallyport/tests/sallyport_enarxcall_tests.rs (/home/ben/code/enarx/target/debug/deps/sallyport_enarxcall_tests-7e9a637e36652c11)
```

Reply:
The output was fine before

Reply:
> The output was fine before

```diff
- Running tests/sallyport_enarxcall_tests.rs
+ Running crates/sallyport/tests/sallyport_enarxcall_tests.rs
```

Reply:
> > The output was fine before
> 
> ```diff
> - Running tests/sallyport_enarxcall_tests.rs
> + Running crates/sallyport/tests/sallyport_enarxcall_tests.rs
> ```

Not sure what this diff is referring to, because that's not the structure that existed before.
Actual diff is:

```diff
- Running crates/sallyport/integration_tests/tests/integration_tests/enarxcall.rs
+ Running crates/sallyport/tests/sallyport_enarxcall_tests.rs
```

Reply:
Actually `cargo` with this PR applied seems to output `Running tests/sallyport_enarxcall_tests.rs` although it should really be `Running crates/sallyport/tests/sallyport_enarxcall_tests.rs`.

This is why @bstrie said:
> I could also suggest that this is a bug in Cargo, and that the first line of that output should read like:

Reply:
Yes, if we didn't change the structure, Cargo would output `Running crates/sallyport/tests/integration_tests/enarxcall.rs` without the need for convoluted filenames

Reply:
We could rename the module to `sallyport_integration_tests` if we want to be explicit about the Sallyport part

Reply:
But regardless, `cargo test syscall` or `cargo test integration_tests` should just work and with the suggested changes it does not